samigo/samigo-app/src/webapp/jsf/delivery/beginTakingAssessment.jsp

```diff
@@ -283,15 +290,29 @@
   </h:commandButton>
 
   <h:panelGroup rendered="#{delivery.sebSetup}">
-    <a id="sebLaunchSeb" class="active">
-      <h:outputText value="#{deliveryMessages.seb_launch_seb}"/>
-    </a>
-    <a id="sebDownloadSeb" class="button" target="_blank">
-      <h:outputText value="#{deliveryMessages.seb_download_seb}"/>
-    </a>
-    <a id="sebDownloadConfiguration" class="button">
-      <h:outputText value="#{deliveryMessages.seb_download_configuration}"/>
-    </a>
+    <h:panelGroup rendered="#{delivery.actionString != 'previewAssessment'}">
+      <a id="sebLaunchSeb" class="active">
+        <h:outputText value="#{deliveryMessages.seb_launch_seb}"/>
+      </a>
+      <a id="sebDownloadSeb" class="button">
+        <h:outputText value="#{deliveryMessages.seb_download_seb}"/>
+      </a>
+      <a id="sebDownloadConfiguration" class="button">
+        <h:outputText value="#{deliveryMessages.seb_download_configuration}"/>
+      </a>
+    </h:panelGroup>
+    <h:panelGroup rendered="#{delivery.actionString == 'previewAssessment'}">
```

Comment:
Why this repetition? Can't we just hide the buttons?

Reply:
I think it's better to repeat the two disabled buttons here, else we yould need to do something like
`<a id="sebLaunchSeb" class="active<h:outputText=" #{devlivery.sebSetup}"">`
I think it's better to disable them then hiding them on the preview, it's closer to what the student sees.

Reply:
Okay, nice